schedule/yast/autoyast_reinstall.yaml

```diff
@@ -2,7 +2,10 @@ name:           autoyast_reinstall
 description:    >
     Parent job produces autoyast profile after successful completion. 
     This test uses generated profile to do autoyast installation.
+vars:
+  AUTOYAST_ADD_REGISTRATION: 1
```

Comment:
Why not to use "FLAVOR" : "Online" instead of add a new variable?

Reply:
because there are other ay test suites with Online medium that do not need to inject it as they use a profile not cloned.